releases/net471/KnownIssues/534719-Networking ServicePoint.ConnectionLimit default behavior with loopback changed unexpectedly.md

```diff
@@ -0,0 +1,73 @@
+# ServicePoint.ConnectionLimit default behavior with loopback changed unexpectedly
+
+## Symptoms
+The limit for HTTP connections per endpoint is controlled by the `ServicePointManager.DefaultConnectionLimit` property.
+This value defaults to 2. The behavior for .NET Framework prior to .NET 4.7.1 was that the limit applied only for 
+non-loopback addresses such as http://www.microsoft.com.  For loopback addresses such as http://localhost, the limit
+for connections was always int.MaxValue (2,147,483,647) unless changed by calling `ServicePoint` class specific APIs.
+
+Due to this bug, when using HttpClient APIs, the connection limit for loopback addresses will now match the limit for
+non-loopback addresses. Thus, the default limit would be 2. This can cause applications to run slower or hang when
+doing multiple, parallel, requests to the http://localhost addresses.
+
+Here is code that can show the problem:
```

Comment:
As long as it's run under 4.7.1, it always shows the problem, no? In that case, can show --> shows

Reply:
Fixed.